app/boards/arm/preonic/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+# SPDX-License-Identifier: MIT
```

Comment:
Please include the copyright header here as well for the ZMK Contributors.

Reply:
Done.